InnerEye/ML/Histopathology/datasets/panda_tiles_dataset.py

```diff
@@ -35,35 +35,36 @@ class PandaTilesDataset(TilesDataset):
     def __init__(self,
                  root: Path,
                  dataset_csv: Optional[Union[str, Path]] = None,
-                 dataset_df: Optional[pd.DataFrame] = None) -> None:
+                 dataset_df: Optional[pd.DataFrame] = None,
+                 occupancy_threshold: Optional[float] = None) -> None:
         super().__init__(root=Path(root) / self._RELATIVE_ROOT_FOLDER,
                          dataset_csv=dataset_csv,
                          dataset_df=dataset_df,
                          train=None)
-
+        if occupancy_threshold is not None:
+            dataset_df_filtered = self.dataset_df.loc[self.dataset_df['occupancy'] > occupancy_threshold]  # type: ignore
+            dataset_df_filtered.reset_index(inplace=True)
```

Comment:
I think here we need to keep the index, because it's already been set. I'd just remove this line.

Reply:
thanks, done